packages/common/src/ui/layout/tables/utils/ColumnDefinitionSetBuilder.ts

```diff
@@ -101,6 +102,7 @@ const getColumnLookup = <T extends RecordWithId>(): Record<
   },
   invoiceNumber: {
     key: 'invoiceNumber',
+    align: ColumnAlign.Right,
```

Comment:
this showed up a problem with the width of the column - the header wasn't in the right place.